src/scenes/Languages/List/LanguageList.tsx

```diff
@@ -14,6 +14,14 @@ const useStyles = makeStyles(({ spacing }) => ({
   options: {
     margin: spacing(3, 0),
   },
+  listContainer: {
+    overflow: 'auto',
+    marginLeft: spacing(-2),
+    padding: spacing(2),
+  },
+  item: {
+    marginBottom: spacing(2),
+  },
```

Comment:
We repeat this a lot. Can we break this out into a simple `ListScrollContainer` or similar? It might be worth moving that new component and `ContentContainer` into a new `components/Layout` directory; what do you think?

Reply:
good idea, implemented.